installation/template/js/remove.js

```diff
@@ -98,6 +98,7 @@ if (document.getElementById('defaultLanguagesButton')) {
           if (successresponse.messages) {
             Joomla.renderMessages(successresponse.messages, '#system-message-container');
           }
+		  Joomla.loadOptions({'csrf.token': successresponse.token});
```

Comment:
```suggestion
          Joomla.loadOptions({'csrf.token': successresponse.token});
```
Same indentation as previous line please.